packages/paste-website/src/pages/components/truncate/index.mdx

```diff
@@ -0,0 +1,193 @@
+---
+title: Truncate
+package: '@twilio-paste/truncate'
+description: The Truncate component can be used to shorten a string of text that overflows a defined area.
+slug: /components/truncate/
+---
+
+import {graphql} from 'gatsby';
+import {Anchor} from '@twilio-paste/anchor';
+import {Box} from '@twilio-paste/box';
+import {Heading} from '@twilio-paste/heading';
+import {Text} from '@twilio-paste/text';
+import {Truncate} from '@twilio-paste/truncate';
+import {Callout, CalloutTitle, CalloutText} from '../../../components/callout';
+import {DoDont, Do, Dont} from '../../../components/DoDont';
+import Changelog from '@twilio-paste/truncate/CHANGELOG.md';
+import {SidebarCategoryRoutes} from '../../../constants';
+
+export const pageQuery = graphql`
+  {
+    allPasteComponent(filter: {name: {eq: "@twilio-paste/truncate"}}) {
+      edges {
+        node {
+          name
+          description
+          status
+          version
+        }
+      }
+    }
+    mdx(fields: {slug: {eq: "/truncate/"}}) {
+      fileAbsolutePath
+      frontmatter {
+        slug
+        title
+      }
+      headings {
+        depth
+        value
+      }
+    }
+  }
+`;
+
+<ComponentHeader
+  name={props.pageContext.frontmatter.title}
+  categoryRoute={SidebarCategoryRoutes.COMPONENTS}
+  githubUrl="https://github.com/twilio-labs/paste/tree/main/packages/paste-core/components/truncate"
+  storybookUrl="https://twilio-labs.github.io/paste/?path=/story/components-truncate--default"
+  data={props.data.allPasteComponent.edges}
+/>
+
+---
+
+<contentwrapper>
+
+<PageAside data={props.data.mdx} />
+
+<content>
+
+## Guidelines
+
+### About Truncate
+
+Truncation, or shortening, is typically used for text or links that exceed the size of their container.
+The Truncate component does all of the heaving lifting for you. You just need to wrap the long
+text with the component, and an element with a set width. This will add cut off the text based on
+the parent containers width, and append an ellipsis `...` at the end.
+
+Good use cases for truncation include:
+
+- Breadcrumbs
+- Long URL links
+- Long item names
+- Long account numbers
```

Comment:
I'd actually remove both these since users may actually need to reference these, and they shouldn't be truncated thoughtlessly. Account numbers usually need to be easily copy-able so they shouldn't be truncated.